src/package.rs

```diff
@@ -441,9 +467,14 @@ impl Package {
         // - 1 tick per rust binary
         // - 1 tick per blob + 1 tick for appending blob dir to archive
         let progress_total = match &self.source {
-            PackageSource::Local { blobs, rust, paths } => {
+            PackageSource::Local {
+                blobs,
+                buildomat_blobs,
+                rust,
+                paths,
+            } => {
                 let blob_work = blobs.as_ref().map(|b| b.len() + 1).unwrap_or(0);
-
+                let buildomat_work = buildomat_blobs.as_ref().map(|b| b.len() + 1).unwrap_or(0);
```

Comment:
nit: only need that extra +1 for blobs as a whole as `add_blobs` only does a single `append_dir_all_async`.